client/README.md

```diff
@@ -49,197 +48,172 @@ for inspection of live javascript to facilitate debugging, use:
 Important Note: The Galaxy repository does not include client script artifacts,
 and these should not be committed.
 
-
-Automatic Rebuilding (Watch Mode)
-=================================
+## Automatic Rebuilding (Watch Mode)
 
 When you're actively developing, it is sometimes convenient to have the client
-automatically rebuild every time you save a file.  You can do this using:
+automatically rebuild every time you save a file. You can do this using:
 
     make client-watch
 
 This will first stage any dependencies (yarn-installed packages like jquery,
 etc), and then will watch for changes in any of the galaxy client source files.
 When a file is changed, the client will automatically rebuild, after which you
-can usually force refresh your browser to see changes.  Note that it is still
+can usually force refresh your browser to see changes. Note that it is still
 recommended to run `make client` after you are finished actively developing
-using `make client-watch`.  
+using `make client-watch`.
 
-Note that there's a new, better option described in the next section.  This
+Note that there's a new, better option described in the next section. This
 method of building will likely be deprecated as HMR is more widely tested.
 
-
-Even Better Automatic Rebuilding (HMR)
-======================================
+## Even Better Automatic Rebuilding (HMR)
 
 For even more rapid development you can use the webpack development server for
-Hot Module Replacement (HMR).  This technique allows swapping out of modules
+Hot Module Replacement (HMR). This technique allows swapping out of modules
 while the application is running without requiring a full page reload most of
 the time.
 
 Setting this up is a little more involved, but it is the fastest possible way
-to iterate when developing the client.  You'll need to start two separate
-processes here.  The first command below starts a special webpack dev server
+to iterate when developing the client. You'll need to start two separate
+processes here. The first command below starts a special webpack dev server
 after a client build, and the second starts a Galaxy server like usual, but
 with extra mappings that redirect client artifact requests to the mentioned
 webpack dev server.
 
-    make client-dev-server
-    GALAXY_CLIENT_DEV_SERVER=1 sh run.sh
+    make client-dev-server GALAXY_CLIENT_DEV_SERVER=1 sh run.sh
```

Comment:
I think these should stay on separate lines?

Reply:
Yeah; vim wrap fail.  Thanks!